python/lsst/ts/ofc/controllers/oic_controller.py

```diff
@@ -292,7 +120,7 @@ def calc_uk_x0(self, mat_f, qx, **kwargs):
         uk : `numpy.ndarray`
             Calculated uk in the basis of degree of freedom (DOF).
         """
-        return -mat_f.dot(qx)
+        return mat_f.dot(qx)
```

Comment:
That's because with the new implementation we use that value and then do `kp * error`, where error is `error = setpoint - state` and in this case `state = mat_f.dot(qx)`